.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+# The name of the workflow.
```

Comment:
What happens if the build or tests fail?

Reply:
Currently nothing, I'd want to add a check that prevents PR with a failed test from merging into the main branch.